apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -158,21 +159,47 @@ config_overlay(Overlay, State) ->
     maybe_create_vtep(Overlay, State),
     maybe_add_ip_rule(Overlay, State).
 
+create_vtep(VXLan, #state{netlink = Pid}) ->
+    #mesos_state_vxlaninfo{
+        vni = VNI,
+        vtep_mac = VTEPMAC,
+        vtep_name = VTEPName
+    } = VXLan,
+    VTEPNameStr = binary_to_list(VTEPName),
+    ParsedVTEPMAC = list_to_tuple(parse_vtep_mac(VTEPMAC)),
+    dcos_overlay_netlink:iplink_add(Pid, VTEPNameStr, "vxlan", VNI, ?VXLAN_UDP_PORT),
+    {ok, _} = dcos_overlay_netlink:iplink_set(Pid, ParsedVTEPMAC, VTEPNameStr).
+
 maybe_create_vtep(_Overlay = #mesos_state_agentoverlayinfo{backend = Backend},
-  #state{netlink = Pid}) ->
+  State=#state{netlink = Pid}) ->
     #mesos_state_backendinfo{vxlan = VXLan} = Backend,
     #mesos_state_vxlaninfo{
-        vni = VNI,
+        vtep_name = VTEPName
+    } = VXLan,
+    VTEPNameStr = binary_to_list(VTEPName),
+    case vtep_present(Pid, VXLan) of
+        false ->
+            lager:debug("~p will be created.", [VTEPNameStr]),
+            create_vtep(VXLan, State);
+        {true, true} ->
+            lager:debug("Attributes of ~p is up-to-date.", [VTEPNameStr]);
+        {true, false} ->
+            lager:info("Attributes of ~p are not update-to-date, and vtep will be recreated.", [VTEPNameStr]),
```

Comment:
could you please change log level to `debug`? we will add a better logging here in `master` branch (it's one of the features dcos-net in dc/os 1.12)